offline/plot.go

```diff
@@ -80,7 +80,7 @@ func computeOptions(def Options, opt ...Options) Options {
 
 var baseHtml = `
 	<head>
-		<script src="https://cdn.plot.ly/plotly-1.58.4.min.js"></script>
+	    <script src="https://cdn.plot.ly/plotly-2.18.1.min.js"></script>
```

Comment:
For some reason I don't understand, when I run `offline.Show` it doesn't work on Google Chrome but it works on Firefox after updating to 2.18.1 :confused: 

how did you test this change?

Reply:
Sorry for the delay, I must have missed your message. 
Interesting, it is running on my Chrome just fine. Version 112.0.5615.49 (Official Build) (x86_64)